samples/openthread/coap_client/boards/nrf52840dk_nrf52840.overlay

```diff
@@ -41,6 +41,9 @@
 &qspi {
 	status = "disabled";
 };
+&mx25r64 {
+	status = "disabled";
+};
```

Comment:
The convention normally is different. node should be disabled by default and then enabled when needed.
```suggestion
&mx25r64 {
	status = "okay";
};
```